rero/rero-ils
# import: add new sources to import from the web

* Adds Library of Congress catalog.
* Adds DNB catalog.
* Adds SLSP catalog.
* Adds UGent catalog.
* Adds KULeuven catalog.
* Adds import record serializers factory.
* Fixes import subtype facets.
* Updates unimarc mapping for BNF import (Tag 464)
* Closes #2065.
* Closes #1825.

Co-Authored-by: Benoit Erken <erken.benoit@gmail.com>
Co-Authored-by: Laurent Dubois <laurent.dubois@itld-solutions.be>
Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):  

* rero/rero-ils-ui#705

## How to test?

Default searches are "anywhere": Example
- covid-19 
- samuelson paul anthony
- 9782492821004

But you can specify key search to get specific results: (key:relation:search term)
- title:all:covid-19
- author:all:samuelson
- isbn:all:9782492821004

When the MARC records contains a 100/700 $0 with idref or gnd identifier of authority, it's automatically linked
to the MEF authority. You should see it in hyperlink in the editor.
It's also uses viafID provided in $0 of LoC records to link the author to a MEF entry


## Code review check list
